### PR TITLE
Fix/119465 attendee registration ux

### PR DESCRIPTION
**Pull request by paulmskim**
https://central.tri.be/issues/ 119465

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/384667ffe88f62656de233ad9c42ae45f76a700e/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/384667ffe88f62656de233ad9c42ae45f76a700e/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:132](https://github.com/moderntribe/event-tickets/blob/384667ffe88f62656de233ad9c42ae45f76a700e/src/Tribe/Editor/Attendee_Registration.php#L132)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Editor/Attendee_Registration.php [hunk 1]
```diff
@@ -87,7 +87,12 @@ public function admin_init() {
 	 * @return string success message
 	 */
 	public function success_notice() {
-		return '<div class="success"><p>' . __( 'Attendee Registration fields saved.', 'event-tickets' ) . '</p></div>';
+		$link    = '<a href="' . get_edit_post_link( $this->post->ID, 'raw' ) . '">' . __( 'return to the content editor', 'event-tickets' ) . '</a>';
+		$notice  = '<div class="success"><p>';
+		$notice .= sprintf( __( 'Attendee Registration fields saved. Make additional changes or %s', 'event-tickets' ), $link );
```
Comment (mitogh) [inline src/Tribe/Editor/Attendee_Registration.php]: I think we should replace `%s` with `%1$s` as provides more flexibility when translating the string the link can be moved around.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/109b524bcd05dd5bfb5682effcb5e166a9968a5d/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/109b524bcd05dd5bfb5682effcb5e166a9968a5d/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:132](https://github.com/moderntribe/event-tickets/blob/109b524bcd05dd5bfb5682effcb5e166a9968a5d/src/Tribe/Editor/Attendee_Registration.php#L132)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *G19.01*, but this was pull requested against *release/M19.01*.

File: src/Tribe/Editor/Attendee_Registration.php [hunk 2]
```diff
@@ -124,7 +129,7 @@ public function render() {
 					$meta->accordion_content( $this->post->ID, $this->ticket_id );
 					?>
 				</div>
-				<button class="button-primary" type="submit">Save</button>
+				<button class="button-primary" type="submit"><?php echo __( 'Save Changes', 'event-tickets' ) ?></button>
```
Comment (sc0ttkclark) [inline src/Tribe/Editor/Attendee_Registration.php]: This should have escaping

```suggestion
				<button class="button-primary" type="submit"><?php esc_html_e( 'Save Changes', 'event-tickets' ) ?></button>
```

File: src/Tribe/Editor/Attendee_Registration.php [hunk 3]
```diff
@@ -115,7 +120,7 @@ public function render() {
 		</style>
 
 		<div id="poststuff"><div class="inside postbox">
-			<a href="<?php echo get_edit_post_link( $this->post->ID, 'raw' );?>">&laquo; Back to Ticket Editor</a>
+			<a href="<?php echo get_edit_post_link( $this->post->ID, 'raw' );?>">&laquo; <?php echo __( 'Back to Editor', 'event-tickets' ) ?></a>
```
Comment (sc0ttkclark) [inline src/Tribe/Editor/Attendee_Registration.php]: This should have escaping on the URL and the text

```suggestion
			<a href="<?php echo esc_url( get_edit_post_link( $this->post->ID, 'raw' ) ); ?>">&laquo; <?php esc_html_e( 'Back to Editor', 'event-tickets' ) ?></a>
```

File: src/Tribe/Editor/Attendee_Registration.php [hunk 4]
```diff
@@ -87,7 +87,12 @@ public function admin_init() {
 	 * @return string success message
 	 */
 	public function success_notice() {
-		return '<div class="success"><p>' . __( 'Attendee Registration fields saved.', 'event-tickets' ) . '</p></div>';
+		$link    = '<a href="' . get_edit_post_link( $this->post->ID, 'raw' ) . '">' . __( 'return to the content editor', 'event-tickets' ) . '</a>';
+		$notice  = '<div class="success"><p>';
+		$notice .= sprintf( __( 'Attendee Registration fields saved. Make additional changes or %1$s', 'event-tickets' ), $link );
```
Comment (sc0ttkclark) [inline src/Tribe/Editor/Attendee_Registration.php]: Should have escaping

```suggestion
		$notice .= sprintf( esc_html__( 'Attendee Registration fields saved. Make additional changes or %1$s', 'event-tickets' ), $link );
```

File: src/Tribe/Editor/Attendee_Registration.php [hunk 5]
```diff
@@ -87,7 +87,12 @@ public function admin_init() {
 	 * @return string success message
 	 */
 	public function success_notice() {
-		return '<div class="success"><p>' . __( 'Attendee Registration fields saved.', 'event-tickets' ) . '</p></div>';
+		$link    = '<a href="' . get_edit_post_link( $this->post->ID, 'raw' ) . '">' . __( 'return to the content editor', 'event-tickets' ) . '</a>';
```
Comment (sc0ttkclark) [inline src/Tribe/Editor/Attendee_Registration.php]: Should have escaping on the link and text

```suggestion
		$link    = '<a href="' . esc_url( get_edit_post_link( $this->post->ID, 'raw' ) ) . '">' . esc_html__( 'return to the content editor', 'event-tickets' ) . '</a>';
```

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/ed9687d9dcfbd5e7964cc1c2622462b46998515e/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/ed9687d9dcfbd5e7964cc1c2622462b46998515e/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:132](https://github.com/moderntribe/event-tickets/blob/ed9687d9dcfbd5e7964cc1c2622462b46998515e/src/Tribe/Editor/Attendee_Registration.php#L132)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/e0063d891a536cb61124703dcc34acc3a4079e0f/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:123](https://github.com/moderntribe/event-tickets/blob/e0063d891a536cb61124703dcc34acc3a4079e0f/src/Tribe/Editor/Attendee_Registration.php#L123)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Attendee_Registration.php:132](https://github.com/moderntribe/event-tickets/blob/e0063d891a536cb61124703dcc34acc3a4079e0f/src/Tribe/Editor/Attendee_Registration.php#L132)

> via jenkins/codesniffer-phpcs

**Pull request merged**